WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -353,3 +361,20 @@ private extension PaymentGatewayAccount {
         .init(rawValue: status)
     }
 }
+
+private extension CardPresentPaymentsPlugins {
+    // This was moved from Yosemite so it can check the feature flag
+    // In a future iteration, we might want to store all the configuration in the WooCommerce layer
+    var supportedCountryCodes: [String] {
```

Comment:
I'm not 100% happy with moving this here, but we can work out a better solution in #5974